graylog2-web-interface/src/components/common/EntityDataTable/EntityDataTable.tsx

```diff
@@ -221,6 +222,22 @@ const EntityDataTable = <Entity extends EntityBase>({
     }));
   }, []);
 
+  /*
+  React.useEffect(() => {
+    const handleOverflow = (e: React.BaseSyntheticEvent) => {
+      console.log(e.target);
+      const scrollContainer = document.getElementById('scroll-container');
+      scrollContainer.style.overflow = scrollContainer.style.overflow !== 'visible' ? 'visible' : 'auto';
+    };
+
+    document.addEventListener('click', handleOverflow, false);
+
+    return () => {
+      document.removeEventListener('click', handleOverflow, false);
+    };
+  }, []);
+  */
```

Comment:
Let's remove the comment.

Reply:
Thanks, I forgot to get that out when I was trying things to fix the overflow issue